src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -240,6 +240,19 @@ MavlinkReceiver::handle_message(mavlink_message_t *msg)
 		handle_message_obstacle_distance(msg);
 		break;
 
+	case MAVLINK_MSG_ID_TUNNEL: {
+			// Re-publish MAVLINK message to uORB only if I want to record it
+			// (parameter MAV_DUMP_TUNNEL is enabled)
+			int32_t mav_dump_tunnel = 0;
+			param_get(param_find("MAV_DUMP_TUNNEL"), &mav_dump_tunnel);
```

Comment:
`param_find()` is heavy function, its does long for loop.
using it on periodical function is not recommended. you should use `DEFINE_PARAMETERS `section

Reply:
anyway, why not to enable/disable the logging by new bit on 'SDLOG_PROFILE'? that seems more sutiable

Reply:
@BazookaJoe1900 Thanks for this information. I used section `DEFINE_PARAMETERS`

I did it in the style of enabling of the GPS dump. Should I fix it there? If you want, I'll do it in the next PR after merging this.

Reply:
My opinion is that it should be added as bit on SDLOG_PROFILE, and not on MAV_DUMP_TUNNEL.
its better to do it under this pr, and not in another one.

Reply:
I concur, mavlink republishing `mavlink_tunnel` at all times is cheap enough to not worry about it. Please extend `SDLOG_PROFILE` instead and we'll finally get this merged.